sklearn/covariance/graph_lasso_.py

```diff
@@ -461,8 +461,8 @@ class GraphLassoCV(GraphLasso):
         values of alphas are passed.
 
     cv : cross-validation generator, optional
-        see sklearn.cross_validation module. If None is passed, defaults to
-        a 3-fold strategy
+        see sklearn.model_selection.split module.
```

Comment:
I'm not terribly happy about this way of documenting `cv` parameters. I would definitely not have a "see somewhere else" as the first thing. I'd also link to the user guide for CV. Here's a rough suggestion (stratified should only be said when it's the case)
```
Determines the cross-validation splitting strategy. It can be:
  - integer, the number of folds in a `(Stratified)KFold`,
  - None, to use (stratified) 3-fold,
  - An object to be used as a cross-validation generator.

See the :mod:`sklearn.model_selection.split` module and :ref:`cross-validation documentation <_cross_validation>`
```